src/plays/text-avatar/components/Avatar/style.scss

```diff
@@ -0,0 +1,10 @@
+.avatar {
```

Comment:
Can we scope this class by adding a play name prefix?

Reply:
you mean `text-avatar_avatar`?

Reply:
Yes, something like that.